index.php

```diff
@@ -0,0 +1,24 @@
+<?php
+$db = new PDO('mysql:host=db; dbname=damian-collection', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$query = $db->prepare("SELECT * FROM `albums`");
+$query->execute();
+$albums = $query->fetchAll();
+
+$displayalbums = '';
+
+foreach ($albums as $album) {
+    $displayalbums .= '<div class="album" style="padding: 10px;"> This album is called ' . $album['name'] . '<br>' .
+                      'The creator is ' . $album['artist'] . '<br>' .
```

Comment:
It's probably worth wrapping each of these in `<p>` so that you don't have to use `<br>`